sysutils/ondir/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             ondir
+version          0.2.3
+revision         0
+
+categories       sysutils
+platforms        darwin
+license          GLP-2
```

Comment:
I downloaded the archive and looked at the license. It is clearly `GPL-2`. Please update the license line to reflect that.

Reply:
apologies. i originally mistyped the license name and then kept misreading it :(
much appreciated. both helpful support and huge patience ;)
\--
ferdy